stdlib/3/re.pyi

```diff
@@ -76,7 +76,16 @@ if sys.version_info < (3, 7):
     # undocumented
     _pattern_type: type
 
-class error(Exception): ...
+if sys.version_info < (3, 5):
```

Comment:
This branch isn't needed, we don't support 3.4 any more.

Reply:
Fixed